internal/constants/constants.go

```diff
@@ -116,10 +116,13 @@ const DefaultRSABitLength int = 4096
 // ExpanderMaxDepth defines the maximum depth to fully expand a given value.
 const ExpanderMaxDepth = int(10)
 
-// StableBranch is the branch mapped to stable builds
-const StableBranch = "stable"
+// ReleaseBranch is the branch used for release builds
+const ReleaseBranch = "release"
 
-// UnstableBranch is the branch used for unstable builds
+// BetaBranch is the branch used for beta builds
+const BetaBranch = "beta"
+
+// UnstableBranch is the branch used for unstable builds.  Note that this branch is not used anymore
 const UnstableBranch = "unstable"
```

Comment:
This can be dropped

Reply:
Shouldn't we leave it in for a little while in case there are installation with a locked version on the unstable branch?

Reply:
nvmd. They wouldn't be running this state tool... I am going to remove it.